### ISSUE TITLE
Issue with "Cache": Not obvious that `Cache` objects persist.

### ISSUE THREAD

**Issue by georgi0u** [mdn/content#2191]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Cache

#### What information was incorrect, unhelpful, or incomplete?

tl;dr: it wasn't obvious to me that `Cache` objects persist in local storage. 

It'd be clearer if the first line read "The Cache interface provides a **persistent** storage mechanism for Request / Response object pairs that are cached, for example as part of the ServiceWorker life cycle."

Or perhaps that last fragment — "...for example as part of the ServiceWorker life cycle" — could be made more solid. `CacheStorage` is available from window scopes as well, and it's not claimed that when a ServiceWorker is terminated, the cache is always cleaned up. Perhaps it should read, "The Cache interface provides a **persistent** storage mechanism for Request / Response object pairs that are cached **in long lived memory. How long a Cache lives is browser dependent, but a single origin's scripts can typically rely on a previously populated Cache to exist.**"

Reading this document with the false assumption that it's talking about an _in-memory_ cache — i.e. one scoped to a window, cleaned up on refresh — is just plausible enough that the article makes sense, albeit while being confusing. "Why can't one just do this in an object?", "Why does a dev bother using this, when they can guarantee the cache is always around by rolling their own?" Answers to which are made obvious by the fact that this is not an in memory cache, but a disk-backed cache.  

I might just be a bit too naive. It did finally click after one too many things didn't make sense.

#### Specific section or headline?

Introduction.

#### What did you expect to see?

"The Cache interface provides a **persistent** storage mechanism for Request / Response object pairs that are cached, for example as part of the ServiceWorker life cycle."

Or:

"The Cache interface provides a **persistent** storage mechanism for Request / Response object pairs that are cached in long lived memory. How long a Cache lives is browser dependent, but a single origin's scripts can typically rely on a previously populated Cache to exist."

#### Did you test this? If so, how?

N/A. This is documentation.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/cache`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Cache
* GitHub URL: https://github.com/mdn/content/blob/master/files/en-us/web/api/cache/index.html
* Last commit: https://github.com/mdn/content/commit/d5cf56f90ed1268223b96501149a7ca40b276438
* Document last modified: 2021-02-04T10:39:33.000Z

</details>

**Comment by mohitdmak** [comment]
I would like to fix this 👍

**Comment by chrisdavidmills** [comment]
@mohitdmak sure thing. I've assigned you.

**Comment by chrisdavidmills** [comment]
> Thanks, @chrisdavidmills !
> 
> sorry to tag you again but should I also mention my PR here on the issue page after making the PR?

No worries at all. The best thing to do is, somewhere in your PR description, include the text `fixes #2191`. If you do this, it will automatically mention the PR here, and then when the PR is reviewed and merged, this issue will automatically be closed.

**Comment by chrisdavidmills** [comment]
Ah great, I see you've done it already — perfect!

**Comment by mohitdmak** [comment]
Oh alright.
Thanks for the explanation👍

**Comment by wbamberg** [comment]
Fixed by https://github.com/mdn/content/pull/2209. Thanks @mohitdmak !
